clients/unshaded/src/main/java/tachyon/client/block/BlockStoreContext.java

```diff
@@ -162,7 +164,7 @@ public synchronized BlockWorkerClient acquireWorkerClient(String hostname) throw
    * @return a {@link BlockWorkerClient} connected to the worker with the given hostname
    * @throws IOException if no Tachyon worker is available for the given hostname
    */
-  public synchronized BlockWorkerClient acquireWorkerClient(NetAddress address)
+  public BlockWorkerClient acquireWorkerClient(NetAddress address)
```

Comment:
I think we need to make `NetworkAddressUtils#getLocalHostName` synchronized now that this method isn't